packages/api/cms-api/src/page-tree/createPageTreeResolver.ts

```diff
@@ -406,6 +436,31 @@ export function createPageTreeResolver({
             await this.pageTreeService.updateNodePosition(newNode.id, { pos: input.pos as number, parentId: newNode.parentId });
             return this.pageTreeNode(newNode.id); // refetch new version
         }
+
+        @Query(() => SitePreviewHash)
+        getSitePreviewHash(): SitePreviewHash {
```

Comment:
would you mind moving this into it's own resolver class (to avoid this file getting even larger)